src/main/java/com/kodilla/ecommercee/service/CartDbService.java

```diff
@@ -0,0 +1,69 @@
+package com.kodilla.ecommercee.service;
+
+import com.kodilla.ecommercee.domain.Cart;
+import com.kodilla.ecommercee.domain.Order;
+import com.kodilla.ecommercee.domain.Product;
+import com.kodilla.ecommercee.domain.User;
+import com.kodilla.ecommercee.exception.CartNotFoundException;
+import com.kodilla.ecommercee.exception.CartNotFoundWhileCreatingOrderException;
+import com.kodilla.ecommercee.exception.ProductNotFoundException;
+import com.kodilla.ecommercee.exception.UserNotFoundException;
+import com.kodilla.ecommercee.repository.CartRepository;
+import com.kodilla.ecommercee.repository.OrderRepository;
+import com.kodilla.ecommercee.repository.ProductRepository;
+import com.kodilla.ecommercee.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class CartDbService {
+
+    private CartRepository cartRepository;
+    private ProductRepository productRepository;
+    private OrderRepository orderRepository;
+    private UserRepository userRepository;
+
+    public void createCart(Long UserId) throws UserNotFoundException {
+        User user = userRepository.findById(UserId).get();
```

Comment:
We should handle exception here, findById can result in getting null you forgot to handle it.

Reply:
orElseThrow(UserNotFoundException::new)

Reply:
Ofcourse ;)